### ISSUE TITLE
Run file system event loop from a thread created by server socket

### ISSUE THREAD

**Issue by dsaenztagarro** [dsaenztagarro/cpp-guile#3]
Tasks:
- [X] Create basic operations in generic List
- [ ] Read received message packets (tokens)
- [ ] Decouple fsevent implementation from event loop
- [ ] Server socket create thread to run fsevent
- [ ] Destroy thread on server socket exit

Postponed:
- Entry in syslog for each thread message (depends on #6)
